test/d-b-n_smoke/arrays.html

```diff
@@ -176,6 +181,17 @@
                             expect(document.querySelector("ul.primitive_people>li>span").innerHTML).to.equal("Homo erectus");
                             done();
                         }, 100);
+                    });
+                    it('for arrays of primitives, trigger a local change, fire a change event, and EVENTUALLY update dom-bind', function (done) {
```

Comment:
I would add to the description a note that this is a local or remote change, or even separate it to two different `describe`s